proposals/partial-properties.md

```diff
@@ -3,16 +3,28 @@ https://github.com/dotnet/csharplang/issues/6420
 
 ### Grammar
 
-The *property_declaration* grammar [(§14.7.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1471-general) is updated as follows:
+The partial modifier on properties is treated just like any other modifier and can be included in any position in a modifier list.
+
+The property grammar [(§15.7.1)](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/classes.md#1571-general) is updated as follows:
 
 ```diff
-property_declaration
--    : attributes? property_modifier* type member_name property_body
-+    : attributes? property_modifier* 'partial'? type member_name property_body
```

Comment:
reading diffs of diffs can be challenging even in simple cases like this, so consider just reading the old version then the new version to understand what's going on.